ext/taglib_flac_picture/taglib_flac_picture_wrap.cxx

```diff
@@ -3157,6 +3259,8 @@ SWIGEXPORT void Init_taglib_flac_picture(void) {
   rb_define_const(SwigClassPicture.klass, "Illustration", SWIG_From_int(static_cast< int >(TagLib::FLAC::Picture::Illustration)));
   rb_define_const(SwigClassPicture.klass, "BandLogo", SWIG_From_int(static_cast< int >(TagLib::FLAC::Picture::BandLogo)));
   rb_define_const(SwigClassPicture.klass, "PublisherLogo", SWIG_From_int(static_cast< int >(TagLib::FLAC::Picture::PublisherLogo)));
+  rb_define_singleton_method(SwigClassPicture.klass, "type_to_string", VALUEFUNC(_wrap_Picture_type_to_string), -1);
```

Comment:
So now we get both `TagLib::FLAC.picture_type_to_string` and `TagLib::FLAC::Picture.type_to_string`. Maybe we should `%ignore` one of them?